packages/components/tooltip/src/tooltip.tsx

```diff
@@ -295,7 +296,29 @@ export const Tooltip = forwardRef<TooltipProps, "div">(
       () => (isOpen && closeOnScroll ? closeNow() : undefined),
     )
 
-    useEventListener(() => triggerRef.current, "pointerleave", closeWithDelay)
+    useEventListener(
+      () => triggerRef.current,
+      "pointerleave",
+      (e) => {
+        if (e.pointerType !== "touch") closeWithDelay()
+      },
+    )
+
+    useEventListener(
+      () => triggerRef.current,
+      "touchstart",
+      () => {
+        if (isOpen) closeWithDelay()
+        else openWithDelay()
+      },
+    )
```

Comment:
i found waring in console when storybook is up (that occurred before you fixed)
please add option like this.

```suggestion
        useEventListener(
      () => triggerRef.current,
      "touchstart",
      () => {
        if (isOpen) closeWithDelay()
        else openWithDelay()
      },
      {
        passive: true,
      },
    )
```

Reply:
Fixed. Thank you for your commented . https://github.com/yamada-ui/yamada-ui/pull/1386/commits/309c3e9c238a4ab668fb3346b9a5126369fbbd31